lib/pengine/clone.c

```diff
@@ -993,11 +983,11 @@ pe__clone_text(pcmk__output_t *out, va_list args)
             g_list_free(list);
         }
 
-        pe__short_output_text(out, stopped_list, child_text, state, NULL, options);
+        pe__short_output_text(out, stopped_list, state, NULL, options);
         free(stopped_list);
     }
 
-    free(child_text);
+    out->end_list(out);
```

Comment:
I suspect at this point there's no point in having separate pe__short_output_*() and pe__clone_*() for text and html. We can probably get rid of pe__short_output_*() altogether.

Reply:
The pe__short_output_ functions are called by the pe__clone_ functions.  Should I inline what they were doing, or do we just not want that output anymore?

Reply:
I was thinking inlining